### PR TITLE
LRDOCS-6433, review portal-search javadoc w/@review tags

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6433, cleared the @review tags in portal-search. cc @arboliveira @lipusz. Let me know if I should send this somewhere else, and please make sure I didn't butcher the meaning of anything.

### CODE REVIEW

**Comment by lipusz** [comment]
I can also take care of this next week if @BryanEngler you have other things to work on

**Comment by rbohl** [comment]
Bump

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java [hunk 1]
```diff
@@ -30,11 +30,15 @@
 import java.util.stream.Stream;
 
 /**
- * Holds various outcomes returned when a search is performed.
+ * Contains the full response of an executed search, as returned from the search
+ * engine.
  *
- * @author André de Oliveira
+ * <p>
+ * i The exact response format and the contents of the response depends on the
```
Comment (BryanEngler) [inline modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java]: what does the "i" do here?

**Comment by BryanEngler** [comment]
everything looks fine to me, just had the one question about the "i"

File: modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java (hunk 1 above)
Comment (rbohl) [reply modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java]: Oh, that - err, it was to see if you really reviewed this closely enough. Well done: :medal_sports: 

:smirk:

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search-api/src/main/java/com/liferay/portal/search/searcher/SearchResponse.java]: haha. well my speed reading skills are improving...

**Comment by BryanEngler** [comment]
if @lipusz didnt catch anything either, do i just close this pr and you will forward it to the next step?

**Comment by rbohl** [comment]
forwarded here: https://github.com/codyhoag/liferay-portal/pull/140

**Pull request closed without merging**